app/controllers/miq_alert_controller.rb

```diff
@@ -234,6 +229,7 @@ def alert_field_changed
 
   # Get information for an alert
   def show
+    assert_privileges("miq_alert_show")
```

Comment:
Do we really need this? I think this has an implicit check as the feature is in the `#{controller}_#{method}` format.